tools/Sanitizer/Leak.supp

```diff
@@ -0,0 +1,2 @@
+leak:libNeXus
```

Comment:
Could we add a little comment to each to describe what we are suppressing, a bit like [here](https://gist.github.com/martyngigg/a616c0fd74be7890ee889d0047ab2971). It just gives a bit more context to what has been included.